### PR TITLE
Discard after delete

### PR DESCRIPTION
**Pull request by idanshaby**
Changes proposed in this pull request:
Introduce a new feature page - *Discard after delete*.

I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): @idanshaby

This pull request needs review by: @mureinik @mykaul @rollandf @tnisan

### CODE REVIEW

File: source/develop/release-management/features/storage/discard-after-delete.html.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+## Current status
+* [Phase one](#usage) (implemented in vdsm config file) - released in oVirt 4.0.
+* [Phase two](#future-plans) (configurable on engine side) - planned for oVirt 4.1.
+
+
+## General functionality and restrictions
+
+### Under the hood
+A new parameter was added to the *vdsm* configuration file, ***discard_enable***.<br/>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/discard-after-delete.html.md]: Where is this file located?

File: source/develop/release-management/features/storage/discard-after-delete.html.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+When it is *true* in a specific vdsm host, that host will issue a *blkdiscard* command on a logical volume right before removing it. That means that each disk or snapshot that this host removes, automatically triggers  it to inform the storage array that this logical volume's blocks were freed and can be used by other luns in the array.
+
+### Possible consumers of the discarded blocks
+After a disk is removed from a block storage domain and its blocks are discarded, there's no restriction to use it. That includes:
```
Comment (mureinik) [inline source/develop/release-management/features/storage/discard-after-delete.html.md]: What is it in this sentence?

File: source/develop/release-management/features/storage/discard-after-delete.html.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+These two properties are completely independent.<br/>
+To understand it better, let's take a look at the flow of removing a disk from a block domain in *vdsm*:
+
+1. If the disk's *Wipe After Delete* is enabled, then the disk is first wiped. If not, *vdsm* moves to phase 2.
+2. If the host that is going to remove it has its *discard_enable* enabled, then *vdsm* issues a *blkdiscard* command on that disk's logical volumes. If not, it moves to phase 3.
+3. The disk's logical volumes are removed.
+
+
+## Usage
+To make it work, the administrator should set the *discard_enable* property in *vdsm*'s `config.py.in` file to ***true*** and then **restart the *vdsmd* service**.
```
Comment (mureinik) [inline source/develop/release-management/features/storage/discard-after-delete.html.md]: config.py.in? not config.py?

Comment (idanshaby) [inline source/develop/release-management/features/storage/discard-after-delete.html.md]: Actually it's non of them, thanks for noticing.
Users should use the /etc/vdsm/vdsm.conf file.

File: source/develop/release-management/features/storage/discard-after-delete.html.md (hunk 1 above)
Comment (idanshaby) [inline source/develop/release-management/features/storage/discard-after-delete.html.md]: I'll explain it in the usage part.
Technically it was added to the config.py.in file but the user should use the /etc/vdsm/vdsm.conf file.

**Pull request merged**